src/theme/theme.ts

```diff
@@ -0,0 +1,27 @@
+import { DefaultTheme } from 'styled-components'
+
+interface Theme {
+  fonts: {
+    heading: string
+    paragraph: string
+  }
+  colors: {
+    primary: string
+    secondary: string
+    accent: string
+  }
+}
+
+const theme: DefaultTheme = {
```

Comment:
I was going to make my own styled components typescript file and decided against it because I'd rather get into more important parts of the app. Looks like I forgot to replace the `DefaultTheme` interface with my own `Theme` interface.